op-node/rollup/derive/span_channel_out.go

```diff
@@ -76,6 +105,9 @@ func (co *SpanChannelOut) Reset() error {
 	co.rlp[1].Reset()
 	co.lastCompressedRLPSize = 0
 	co.compressed.Reset()
+	if co.compressionAlgo == BROTLI {
+		co.compressed.WriteByte(0b0001)
```

Comment:
0b0001 -> 0x01 would be more clear (that you're writing a full byte with value "1")